src/structures/SkyBlock/SkyblockMember.js

```diff
@@ -33,47 +32,32 @@ class SkyblockMember {
      * Timestamp when player first joined SkyBlock
      * @type {number}
      */
-    this.firstJoinTimestamp = data.m.first_join;
+    this.firstJoinTimestamp = data.m.profile?.first_join;
     /**
      * Timestamp when player first joined SkyBlock as Date
      * @type {Date}
      */
-    this.firstJoinAt = new Date(data.m.first_join);
-    /**
-     * Timestamp when player first joined the SkyBlock Hub
-     * @type {number}
-     */
-    this.firstJoinHubTimestamp = data.m.first_join_hub;
-    /**
-     * Timestamp when player first joined the SkyBlock Hub as Date
-     * @type {Date}
-     */
-    this.firstJoinHubAt = new Date(this.firstJoinTimestamp + data.m.first_join_hub);
-    /**
-     * Last save timestamp
-     * @type {number}
-     */
-    this.lastSaveTimestamp = data.m.last_save;
-    /**
-     * Last save timestamp as Date
-     * @type {Date}
-     */
-    this.lastSaveAt = new Date(data.m.last_save);
+    this.firstJoinAt = new Date(data.m.profile?.first_join);
     /**
      * Last save timestamp
      * @type {number}
      */
-    this.lastDeathTimestamp = data.m.last_death;
+    this.lastDeathTimestamp = data.m.player_data?.last_death;
     /**
      * Last death timestamp as Date
      * @type {Date}
      */
-    this.lastDeathAt = new Date(skyblock_year_0 + data.m.last_death * 1000);
+    this.lastDeathAt = new Date(skyblock_year_0 + data.m.player_data?.last_death * 1000);
     /**
      * Experience
      * @type {number}
      */
     this.experience = data.m.leveling?.experience ?? 0;
+    /**
+     * Skyblock Level
+     * @type {number}
+     */
+    this.level = this.experience / 100 ?? 0;
```

Comment:
what if `this.experience` is 0?

Reply:
`console.log(0 / 100 ?? 0)` Returns `0`?

Reply:
oops mb